Makefile

```diff
@@ -0,0 +1,11 @@
+execute_dev_checks: check_security test check_code_style
```

Comment:
@daria-sieroshtan 
I think this is not your complete workflow.
Consider, for example, deploying project to production. What commands are you going to ysein this case?